build.gradle

```diff
@@ -50,21 +50,37 @@ subprojects {
       javadoc = true
       tests = false
     }
+    nexus {
+      def gitProvider = new nebula.plugin.info.scm.GitScmProvider()
+      def shortHash = gitProvider.calculateChange(new File('.'))
```

Comment:
changed, but what is the gain of this? If we exlcude the nebula plugin we also lose the jgit dependency, right? You just dont want to have code depending on this in the `build.gradle` file?